used html and markdown

nxnode opened the pull request:
does it matter what I use? I just noticed github allowed me to use certain hrml tags. 1 and 2 both are look the same but I wasn't sure if using hrml would have another unintended consequence like everything seems to.

nxnode committed:
used html and markdown

does it matter what I use? I just noticed github allowed me to use certain hrml tags. 1 and 2 both are look the same but I wasn't sure if using hrml would have another unintended consequence like everything seems to.

W-Ely reviewed (commented):
It does matter, README.`md`s are written in Markdown.  At least one reason is that it formats nicely on Github but also when it's cloned down it's still legible in plain text.

nxnode commented:
I figured! That's why I asked. I read in the forums that lots of people use html in their README.md's but github basically only allows html tags that can be used as markup - so that must be due to what you mentioned when cloning down. I will just use markdown but I still wonder if it matters since github only allows html tags that have equivalent markdown, would it translate just fine or be a mess?? 

None of this matters other than making it look more closely to what I actually want it to look like. But I have seriously started to recognize a trend - it's almost always not worth wasting time exploring other ways since they inherently introduce issues.

nxnode committed:
second update with less html

I was using the <details> tag because it was in the official github documentation on markdown but technically that is not markdown. Thoughts? I promise, I am not trying to be contrary! But I almost left the tags in without realizing they were NOT markdown since I got that from the github documentation so I am just curious.

W-Ely commented:
These are good questions!

In short, I'd not mix html and markdown even though virtually every Markdown renderer supports html.  I just don't see it used like that, even though it's supported.  I'm sure if we search it out we can find some, but having spent time in many thousands of repos and not having seen it, I think it would be unconventional.  Rather than collapsing any part of the document and requiring a click to expand, having a table of contents that uses [anchors](https://roachhd.gitbooks.io/master-markdown/content/syntax/anchors.html) might approximate what you are after, while still allowing someone to simply scroll to see the whole thing.

**Another thing to keep in mind is it's kinda simple to write, and can also be used within these description and comment boxes...**

nxnode committed:
removed all HTML tags

And changed the internal anchors to link to the new name test_check_same_case.py
